lib/run/index.js

```diff
@@ -54,8 +54,8 @@ var fs = require('fs'),
      * @type {Object}
      */
     knownReporterErrorMessages = {
-        html: '  run `npm install newman-reporter-html`',
-        teamcity: '  run `npm install newman-reporter-teamcity`'
+        html: '  run `npm install newman-reporter-html`\n',
```

Comment:
lol ... i spotted this sneaky out of context change in this PR. don't make a habit of it. :-p